src/Rpde/RpdeKind.php

```diff
@@ -20,6 +20,7 @@ class RpdeKind
     const HEADLINE_EVENT = "HeadlineEvent";
     const EVENT = "Event";
     const EVENT_SERIES = "EventSeries";
+    const ON_DEMAND_EVENT = "OnDemandEvent";
```

Comment:
@nathansalter it might be worth keeping this one in? Agreed the rest look like a generator bug, so perhaps we create an issue for those in models-lib?

Reply:
Have created https://github.com/openactive/models-lib/issues/49 in fact

Reply:
Have broken out this fix into #86